analysis/paper/paper.Rmd

```diff
@@ -277,32 +292,21 @@ res <- "integrated_snn_res.1.2"
 ePBMC.integrated.obj <- SetIdent(object = ePBMC.integrated.obj, value = res)
 
 # Run UMAP embedding
-ePBMC.integrated.obj <- RunUMAP(ePBMC.integrated.obj, dims = 1:25, min.dist = 0.5)
+ePBMC.integrated.obj <- RunUMAP(ePBMC.integrated.obj,
+  dims = 1:25,
+  min.dist = 0.6,
+  n.neighbors = 75
+)
 
 # Build hierarchical clustering based on clustering results
+DefaultAssay(ePBMC.integrated.obj) <- "RNA"
+ePBMC.integrated.obj <- NormalizeData(ePBMC.integrated.obj)
 ePBMC.integrated.obj <- BuildClusterTree(ePBMC.integrated.obj, assay = "RNA", dims = 1:25)
 ```
 
 Next, we perform additional filtering of doublets based on our knowledge of established immune cell markers. In theory, CD3G+ T cells should not express B cell markers such as MS4A1 or CD19, or immunoglobulins.
 
-```{r Filtering step: Remove putative doublets based on knowledge of immune cell development}
```

Comment:
Were these steps removed? If so, correct the immediately prior markdown text that refers to them

Reply:
Resolved. These steps were removed in the end.